uui-components/src/types.ts

```diff
@@ -0,0 +1 @@
+export type ControlSize = 'none' | '24' | '30' | '36' | '42' | '48';
```

Comment:
we can't move sizes in uui-components, because they can be different between skins